setup.py

```diff
@@ -88,7 +88,6 @@ def run(self):
     "pyparsing==2.4.7",
     "typing_extensions>=3.7.4",
     "fsspec>=0.8.5",
-    "dvclive>=0.0.1a0",
```

Comment:
I guess this was a typo and it was meant for `tests`, right? cc @pared

Reply:
Right, sorry for that